src/riak_kv_pb_timeseries.erl

```diff
@@ -64,6 +64,7 @@
 -define(E_MISSING_TS_MODULE, 8).
 -define(E_DELETE,   9).
 -define(E_GET,     10).
+-define(E_BAD_KEY_LENGTH, 11).
```

Comment:
The thing is, however, is that from the release of Riak TS onward, these codes will have meaning. If the codes change after that point, documentation will have to change, Client Services' understanding of the codes will have to change, etc - there will be a ripple effect.

I think it's a safe bet to choose something like 1001-1250 as "reserved" for timeseries. Best case, the codes will never change in the future and everyone will be happy. Worst case, they will, in which case we would be in the same boat as if they stay 1-11.